runner/execer/os/os_execer.go

```diff
@@ -89,21 +90,35 @@ func (p *osProcess) monitorMem(memCap execer.Memory, stat stats.StatsReceiver) {
 	reportThresholds := []float64{0, .25, .5, .75, .85, .9, .93, .95, .96, .97, .98, .99, 1}
 	memTicker := time.NewTicker(10 * time.Millisecond)
```

Comment:
This should be increased back to a reasonable value as you suggest once we get this working.